requirements.md

```diff
@@ -32,7 +32,7 @@ Req. 2.1 The App shall annotate each output image to show the number correspondi
 
 Req. 2.1.1 Each annotation shall consist of text in 8pt font or greater.
 
-Req. 2.1.2 Each annotation shall be presented in each output image directly above the distict color to which the annotation corresponds.
+Req. 2.1.2 Each annotation shall be presented in each output image directly above the distict color to which the annotation corresponds
```

Comment:
Does "directly above" need to be defined? What if annotation overlap? How far above? How about "Each annotation shall be presented in each output image so that the annotation overlaps the distinct color to which the annotation corresponds"